### DISCUSSION TITLE
Deformation axis FFD

### DISCUSSION THREAD

**Discussion by benoitpaillard** [mdolab/dafoam#706] [General]
Hi all,

When running a case with FFD as design variable, I noticed you get as many design variable as there are points, ie for naca0012 there are 20 points thus you get 20 variables, whereas for 2D cases there should be forty variable, 1 x and 1 y for each point.

In v2 scripts there were options to set x or y as design variables. However I can't find where this is defined in new scripts.

thanks for your help

**Comment by NUAA-qinfei** [comment]
Hi,I know your mean,you can do it like this,I test it ok.
![Screenshot from 2024-11-21 15-41-41](https://github.com/user-attachments/assets/411cb801-4290-4475-b148-47fb572053bb)

**Comment by benoitpaillard** [comment]
thanks a lot, I couldn't find it.
